v3/src/hooks/use-drag-drop.ts

```diff
@@ -29,7 +29,7 @@ export interface IUseDraggableAttribute extends Omit<UseDraggableArguments, "id"
   prefix: string
   attributeId: string
 }
-export const useDraggableAttribute = ({ prefix, attributeId, ...others }: IUseDraggableAttribute) => {
+export const useDraggableAttribute = ({ prefix, attributeId, disabled, ...others }: IUseDraggableAttribute) => {
```

Comment:
>Passes disabled property specifically into the useDraggableAttribute

This removes the `disabled` property from the props so that it is _never_ passed to the underlying `useDraggable`, i.e. it means that it's always enabled. The `disabled` flag is used to disable column header drag while the menu is open. I think the proper fix will require understanding why the `disabled` property is getting the incorrect value rather than by eliminating the function of the `disabled` property altogether.